qt/python/instrumentview/instrumentview/FullInstrumentViewModel.py

```
@@ -361,29 +361,58 @@ def update_point_picked_detectors(self, index: int, expand_to_parent_subtree: bo
+
+    def _build_detector_info_list(
+        self,
+        ws_indices: np.ndarray,
+        det_ids: np.ndarray,
+        xyz_positions: np.ndarray,
+        spherical_positions: np.ndarray,
+        counts: np.ndarray,
+    ) -> list[DetectorInfo]:
+        return [
+            DetectorInfo(det.getName(), det_id, ws_index, xyz, spherical, det.getFullName(), int(count))
+            for ws_index, det_id, xyz, spherical, count in zip(ws_indices, det_ids, xyz_positions, spherical_positions, counts)
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)